sycl/doc/extensions/README.md

```diff
@@ -7,32 +7,23 @@ DPC++ extensions status:
 | [FPGA reg](IntelFPGA/FPGAReg.md)                                                                                            | Supported(OpenCL: ACCELERATOR)            | |
 | [SYCL_INTEL_kernel_restrict_all](KernelRestrictAll/SYCL_INTEL_kernel_restrict_all.asciidoc)                                 | Supported(OpenCL)                         | |
-| [SYCL_INTEL_attribute_style](KernelRHSAttributes/SYCL_INTEL_attribute_style.asciidoc)                                       | Proposal                                  | |
-| [Queue Order Properties](OrderedQueue/OrderedQueue_v2.adoc)                                                                 | Supported                                 | |
-| [Queue Shortcuts](QueueShortcuts/QueueShortcuts.adoc)                                                                       | Supported                                 | |
 | [Reductions for ND-Range Parallelism](Reduction/Reduction.md)                                                               | Partially supported(OpenCL: CPU, GPU; CUDA) | Not supported: multiple reduction vars, multi-dimensional reduction vars |
-| [SYCL_INTEL_relax_standard_layout](RelaxStdLayout/SYCL_INTEL_relax_standard_layout.asciidoc)                                | Supported                                 | |
 | [SYCL_INTEL_reqd_work_group_size](ReqdWorkGroupSize/SYCL_INTEL_reqd_work_group_size.asciidoc)                               | Supported(OpenCL: CPU, GPU)               | |
 | [SPV_INTEL_function_pointers](SPIRV/SPV_INTEL_function_pointers.asciidoc)                                                   | Supported(OpenCL: CPU, GPU; HOST)         | |
 | [SPV_INTEL_inline_assembly](SPIRV/SPV_INTEL_inline_assembly.asciidoc)                                                       | Supported(OpenCL: GPU)                    | |
 | [LocalMemory](LocalMemory/LocalMemory.asciidoc)                                                                             | Supported(OpenCL; CUDA)                   | Revision 1 of the spec is fully supported, future revisions are expected to expand the functionality |
 | [SYCL_INTEL_static_local_memory_query](StaticLocalMemoryQuery/SYCL_INTEL_static_local_memory_query.asciidoc)                | Proposal                                  | |
 | [Sub-groups for NDRange Parallelism](SubGroupNDRange/SubGroupNDRange.md)                                                    | Deprecated(OpenCL: CPU, GPU)              | |
 | [Sub-groups](SubGroup/SYCL_INTEL_sub_group.asciidoc)                                                                        | Deprecated                                | |
-| [SYCL_INTEL_unnamed_kernel_lambda](UnnamedKernelLambda/SYCL_INTEL_unnamed_kernel_lambda.asciidoc)                           | Supported(OpenCL)                         | |
-| [Unified Shared Memory](USM/USM.adoc)                                                                                       | Supported(OpenCL)                         | |
```

Comment:
Would it make sense to have a table for removed/promoted extensions? It may be an unlikely case, but someone could come here looking for an extension they have seen before but find nothing because it has been removed. Knowing that the extension was promoted to a newer SYCL version could at least redirect them to the spec or other resources.

Reply:
I'm a little hesitant to do this because it's unclear whether these older extension document really described the behavior of any previous DPC++ release.  I think a lot of them were proposals, and those proposals got tweaked as they were promoted into the SYCL specification.  As a result, the extension spec doesn't always match the SYCL spec.  However, it's not clear if any DPC++ release ever implemented the wording in the extension spec, so it's not clear how useful it is to keep that old wording around.  In fact, I'm worried that keeping the old wording around could cause confusion.

I think our extension process is getting a lot more formal, so we will avoid this problem in the future.  With the new process, it will be easy to tell which extensions are implemented vs. proposed.  In addition, extensions will always be implemented in their own namespace (not directly in the `sycl` namesapce), so we can keep both versions around if an extension is later promoted into the core SYCL spec.  This will all make it easier to maintain a list of deprecated extensions that have since been promoted into the core spec.

For these older extensions, though, I'd prefer to delete the ones that have been promoted.  If we think it's useful, we could have a table with just the name of the extension and a note saying that it was promoted to SYCL 2020.  I'm not sure how useful this is, though.

Reply:
> If we think it's useful, we could have a table with just the name of the extension and a note saying that it was promoted to SYCL 2020. I'm not sure how useful this is, though.

That was the kind of table I was thinking. I agree, we shouldn't keep the old - potentially outdated - extensions/proposals around after promotion, but a table that says something along the lines of "This was promoted in SYCL 2020. See ..." could potentially be useful in preserving at least some visible history. That said, if you don't think it makes much sense I am okay with these changes as-is.

Reply:
I did this in db2c15e1d347d3fd90ffb5c5d27a24b6c3c93842